bd/checkUserAlreadyExists.go

```diff
@@ -0,0 +1,26 @@
+package bd
+
+import (
+	"context"
+	"time"
+
+	"github.com/SaiKumarMalve/Gator-News/models"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func CheckUserAlreadyExists(email string) (models.User, bool, string) {
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
+
+	db := MongoCN.Database(("Gator-News"))
```

Comment:
DB name is incorrect and also the collection name is all lower case letters

Reply:
I have updated the code. Please review.